src/api.coffee

```diff
@@ -127,6 +128,9 @@ start = ->
     .then (results) ->
       TaskActions.loadedUserTasks(results.items)
 
+  apiHelper TeacherTaskPlanActions, TeacherTaskPlanActions.load, TeacherTaskPlanActions.loaded, 'GET', (course_id) ->
+    url: "/api/courses/#{course_id}/plans.json"
```

Comment:
I noticed what you had to do here because you couldn't create a `/plans` file. #51 should fix that little oversight on my part :blush: